tasks/configure_gitlab_ce.yml

```diff
@@ -1,5 +1,19 @@
 ---
 
+- name: Add Yarn repository
+  apt_repository:
+    repo: '{{ gitlab__yarn_apt_repo }}'
+    state: 'present'
+    update_cache: True
+  when: gitlab_version | version_compare("8.17", operator="ge", strict=True)
+
+- name: Install Yarn
+  apt:
+    name: 'yarn'
+    state: 'latest'
```

Comment:
This should be set to `present` so that the role does not upgrade the packages on a subsequent run.